StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -95,6 +97,19 @@ class RowButton: UIView {
         } else {
             self.sublabel = nil
         }
+
+        if showDefaultPMBadge {
+            let defaultBadge = UILabel()
+            defaultBadge.font = appearance.scaledFont(for: appearance.font.base.medium, style: .footnote, maximumPointSize: 20)
```

Comment:
Figma shows in vertical mode this being a weight of regular and caption1 for style.